generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -14,6 +18,26 @@ sed -i.bak 's/canvas: None,/canvas: Some("#bevy".to_string()),/' crates/bevy_win
 # setting the asset folder root to the root url of this domain
 sed -i.bak 's/asset_folder: "assets"/asset_folder: "\/assets\/examples\/"/' crates/bevy_asset/src/lib.rs
 
+export CARGO_TARGET_DIR="target"
+
+if [ "$1" = --ci ] ; then
+    # Do not optimize on --ci check,
+    # wasm-opt takes A LOT of time to run
+    wasm_opt_flag="-O0"
```

Comment:
Note: I set the flag to `-O0`, because this allows validating the `wasm-opt` command in CI, reducing divergence between ci and deploy, and avoiding nasty surprises when deploying for the first time